fastapi/utils.py

```diff
@@ -21,6 +21,18 @@
 def is_body_allowed_for_status_code(status_code: Union[int, str, None]) -> bool:
     if status_code is None:
         return True
+    if status_code in [
```

Comment:
it'd be more efficient to have a set here instead of list?

Reply:
That would be also more consistent below where its also doing set-based check. So here using instead:
```python
    # https://swagger.io/docs/specification/describing-responses/#default
    # https://swagger.io/docs/specification/describing-responses/#status-codes
    if status_code in {
        "default",
        "2XX",
        "2xx",
        "3XX",
        "3xx",
        "4XX",
        "4xx",
        "5XX",
        "5xx",
    }:
        return True
```
Also adding a comment where these come from would be a good idea.